docs/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "tabWidth": 4,
```

Comment:
How would this get picked up by the editor? Should we be setting the tool as the default over VSCode settings?

Reply:
These are configuration pieces for the Prettier tool, available in the VSCode marketplace. You are welcome to use whatever formatting tool you prefer, but please wrap at 100 columns.  VSCode will do "visual" wrapping at 100 characters, but will not inject linebreaks by default unless you add some code-formatting tool that does it.

Reply:
Do we want to mention MarkdownLint for VSCode authors or other tooling suggestions?